File: doom.cpp
#include <iostream>
#include <iomanip>
#include <string>

int main() {
    std::string tipo_demonio;
    double distancia;

    // Leer el tipo de demonio y la distancia desde la entrada
    std::getline(std::cin, tipo_demonio);
    std::cin >> distancia;

    double velocidad = 0;
    double tiempo = 0;

    // Determinar la velocidad del demonio y calcular el tiempo
    if (tipo_demonio == "Zombieman") {
        velocidad = 5;
    } else if (tipo_demonio == "Imp") {
        velocidad = 10;
    } else if (tipo_demonio == "Baron of Hell") {
        velocidad = 3;
    } else {
        std::cout << "Tipo de demonio no válido" << std::endl;
        return 0; // Salir del programa si el tipo de demonio no es válido


        
    }

    tiempo = distancia / velocidad;

    // Imprimir el resultado con dos decimales de precisión
    std::cout << std::fixed << std::setprecision(2) << "Tiempo: " << tiempo << " segundos" << std::endl;

    return 0;
}

File: Doom.py
#Lee el tipo de demonio
tipo_demonio = input() #definimos las variables /entrada

#Lee la distancia
distancia = float(input())# definimos las variables/entrada

#Calcula el tiempo/ proceso que hara el programa 
if tipo_demonio == "zombieman":
    velocidad = 5
    tiempo = distancia / velocidad
    print("Tiempo: {:.2f} segundos".format(tiempo))#me toco averiguar como se usa el formato tiempo 
elif tipo_demonio == "imp":
    velocidad = 10
    tiempo = distancia / velocidad #en esta parte se me dificulto saber que deberia usar
    print("Tiempo: {:.2f} segundos".format(tiempo))
elif tipo_demonio == "baron of hell":
    velocidad = 3
    tiempo = distancia / velocidad
    print("Tiempo: {:.2f} segundos".format(tiempo))
else:
    print("Tipo de demonio no válido")
    #Espera una entrada del usuario antes de cerrar la terminal
input("Presiona Enter para salir")
